web/client/plugins/CRSSelector.jsx

```diff
@@ -40,7 +41,9 @@ class Selector extends React.Component {
         typeInput: PropTypes.func,
         enabled: PropTypes.bool,
         currentBackground: PropTypes.object,
-        onError: PropTypes.func
+        onError: PropTypes.func,
+        AllowedUsers: PropTypes.array,
```

Comment:
The convention is that capital letters are for classes. 
The filter is on Roles, not on Users, so please use `allowedRoles` (roles, not capitalized) instead